Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -92,7 +92,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockPTechFilled
-  cost: 800
+  cost: 1200
```

Comment:
Seems like not related to PR, what is this?

Reply:
It's the price for the restock crate. If it's not set higher than the value of the vending machine contents, then cargo gets an infinite money glitch and it will fail tests.

Reply:
Why it is in PR that not related to cargo...? you didn't described this change, this why i asking.
"Add encryption keys to the PTech" is not "Fixing infinite money glitch in cargo"

Reply:
Because you need to change the price of the restock boxes when you change the contents of a vending machine.
```
# If you've arrived here after a failed NoCargoOrderArbitrage test, it's likely
# because the product in question contains a restock box which links to a
# vending machine inventory that has had items added to it recently and caused
# its calculated price to exceed the cost below.
#
# The costs will need to be kept in line with the inventory of the respective
# vending machines. An extra hundred or two profit margin should be fine.
````

Reply:
Aaaaa, wawawa, I just understood that this is a package for restock le encryption key vendor, sorry